workloads/logging/README.md

```diff
@@ -0,0 +1,67 @@
+# Logging
```

Comment:
do we also need a smoke test?

Reply:
We wouldn't need a smoke test since the script is around deploying the logging stack before running the workload IMHO. For the CI, we can call it as is with the defaults, it should just take around 3-4 mins. Thoughts?

Reply:
Agreed after more thought... The smoke test can be for when the workload is running.